content-resources/src/main/java/flyway/downloadbasket/V1_00_1__add_download_basket_to_views.java

```diff
@@ -0,0 +1,49 @@
+package flyway.downloadbasket;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import fi.nls.oskari.map.view.ViewService;
+import fi.nls.oskari.map.view.ViewServiceIbatisImpl;
+
+public class V1_00_1__add_download_basket_to_views implements JdbcMigration {
+	
+	private static final ViewService VIEW_SERVICE = new ViewServiceIbatisImpl();
```

Comment:
Adding to user views in addition to default view means that any personalized default views will get this bundle as well as the system default.